starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -69,19 +68,17 @@ public Inventory create(Inventory inventory) {
   public Optional<Inventory> retrieve(String id) {
     Inventory ret = mongoTemplate.findById(id, Inventory.class);
 
-    //built-in null handling
     return Optional.ofNullable(ret);
   }
 
   /**
    * Update Inventory.
-   * @param id Inventory id to Update.
    * @param inventory Inventory to Update.
    * @return Updated Inventory.
    */
-  public Optional<Inventory> update(String id, Inventory inventory) {
-    // TODO
-    return Optional.empty();
+  public Inventory updateMe(Inventory inventory) {
```

Comment:
- Change name back to update
- Should return an Optional<Inventory> not just an Inventory
- should be
return Optional.ofNullable(this.mongoTemplate.save(inventory));